docs/deploy-options.md

```diff
@@ -2,12 +2,19 @@
 
 The [README][0] shows you a simple way to get started with Contour on your cluster. This topic explains the details and shows you additional options.
 
+## Manual and Specialized Deployments
+
+This document outlines how to run Contour using a Kubernetes Service of `Type: LoadBalancer`.  If you don't have a cluster with that capability or if you don't want to use it you have two other options:
+
+* Use a Kubernetes Service with `Type: NodePort`.  This configure every node in your cluster to forward traffic on that Node to Contour.
```

Comment:
This one is also documented below (##Running without a Kubernetes LoadBalancer). Do we need it in both places? The more detailed version is more helpful, too.
But if you prefer this (shorter) version, it still needs minor edits:
"This Service configures every node in your cluster to forward traffic on the node to Contour." (the generic reference to "node")
OR
"This Service configures every Node in your cluster to forward traffic on the Node to Contour." (the specific object reference. consistency ...)

Reply:
Merging this into section below.